ros2launch/ros2launch/command/launch.py

```diff
@@ -103,6 +102,26 @@ def add_arguments(self, parser, cli_name):
             help="Arguments to the launch file; '<name>:=<value>' (for duplicates, last one wins)")
         arg.completer = SuppressCompleterWorkaround()
 
+        sec_args = parser.add_argument_group(title='Security', description='Security related args')
+        arg = sec_args.add_argument(
+            '--secure',
```

Comment:
This flag is a bit of a double whammy since it does two things: it enables security if present, and if a positional argument follows it assumes it is the keystore. We can separate these into two arguments if preferred, this just reduces the need for extra flags.

Reply:
I suspect most users won't actually supply an argument here unless they're debugging something, in which case we need an arg that makes sense without one (or two args, as @Arnatious says). I think `--secure` is the best fit for a single arg that makes sense as a value-less flag, but we can _make_ is a flag add a second `--keystore` arg that requires a value. We would just need to figure out what we should do if a keystore is provided without `--secure`. I prefer the current UX, personally.

Reply:
Thanks for the clarification. I think leaving it as `--secure` makes sense.